### PR TITLE
Fix error suppression pragmas in bluebird libdefs

### PR DESCRIPTION
**Pull request by julienw**
I just fixed the error suppression pragmas and made the tests pass.

### CODE REVIEW

File: definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js [hunk 1]
```diff
@@ -36,22 +36,22 @@ let mapSeriesResult: Promise<number[]> = Bluebird.mapSeries([1], mapper);
...
-  // $FlowExpectedError Property not found in string
+  // $FlowExpectedError[prop-missing] Property not found in string
   responseBody.then(r => r);
 });
 
 Bluebird.all([1, Bluebird.resolve(1), Promise.resolve(1)]).then(function(r: Array<number>) { });
 Bluebird.all(['hello', Bluebird.resolve('world'), Promise.resolve('!')]).then(function(r: Array<string>) { });
 Bluebird.all(['hello', Bluebird.resolve('world'), Promise.resolve('!')]).then(function(r: [string, string, string]) { });
 
-// $ExpectError Wrong tuple type
```
Comment (julienw) [inline definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js:54]: This was one is the old one.

File: definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js [hunk 2]
```diff
@@ -78,10 +78,9 @@ fooPromise(1, 'b').catch(function(e) {
 fooPromise(1, 'b').catch(Error, function(e: Error) {
   let m: string = e.message;
 });
-// $FlowExpectedError
-fooPromise(1, 'b').catch(Error, function(e: NetworkError) {
```
Comment (julienw) [inline definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js:82]: I believe this wasn' tthe right test: `NetworkError` doesn't exist and this was failing for the wrong cause.

**Review by Brianzchen — APPROVED** [review]

**Review by pascalduez — APPROVED** [review]
Thanks!

**Comment by pascalduez** [comment]
Thanks for the review @Brianzchen !

**Pull request merged**